### PR TITLE
Fix broken live examples in `zh-cn/learn/css`

### PR DESCRIPTION
**Pull request by dukecat0**
Fix broken live examples in `zh-cn/learn/css`.

### CODE REVIEW

**Review by t7yang — CHANGES_REQUESTED** [review]
1. Do not change `id` or modify hidden content (or something else) which do not match what you declare — Fix broken live examples in `zh-cn/learn/css`:
  - All `id` will be remove after convert from html to markdown format.
  - Hidden contents will remove in the future.
2. Please make id of `EmbedLiveSample` same as the latest English version.

**Comment by meowmeowmeowcat** [comment]
@t7yang However, will live samples work if we don't change `id` or modify hidden content?

**Comment by meowmeowmeowcat** [comment]
I'll change `id` so id of `EmbedLiveSample` will be the same as the latest English version.

File: files/zh-cn/learn/css/css_layout/positioning/index.html [hunk 1]
```diff
@@ -539,8 +541,9 @@ <h6 id="Sticky_positioning_example">Sticky positioning example</h6>
 }
 </pre>
 
+<div id="Sticky_2">
 <div class="hidden">
-<pre class="notranslate">body {
+<pre class="brush:css notranslate">body {
```
Comment (meowmeowmeowcat) [inline files/zh-cn/learn/css/css_layout/positioning/index.html:546]: I've added `brush:css` here. If this is missing, then the live sample won't work.

File: files/zh-cn/learn/css/css_layout/positioning/index.html [hunk 2]
```diff
@@ -557,7 +560,7 @@ <h6 id="Sticky_positioning_example">Sticky positioning example</h6>
 }
 </pre>
 
-<pre class="notranslate">&lt;h1&gt;Sticky positioning&lt;/h1&gt;
+<pre class="brush:html notranslate">&lt;h1&gt;Sticky positioning&lt;/h1&gt;
```
Comment (meowmeowmeowcat) [inline files/zh-cn/learn/css/css_layout/positioning/index.html:563]: I've added `brush:html` here. If this is missing, then the live sample won't work.

**Comment by meowmeowmeowcat** [comment]
> Do not change id or modify hidden content (or something else) which do not match what you declare — Fix broken live examples in zh-cn/learn/css

I've changed some `id` to match with `EmbedLiveSample`. (And also follow the latest English version)

File: files/zh-cn/learn/css/css_layout/grids/index.html [hunk 3]
```diff
@@ -83,6 +83,7 @@ <h3 id="定义一个网格">定义一个网格</h3>
 
 <p>在规则里加入你的第二个声明。刷新页面后，你会看到子项们排进了新定义的网格中。</p>
 
+<div id="Grid_1">
```
Comment (t7yang) [inline files/zh-cn/learn/css/css_layout/grids/index.html:86]: Please make sure for every added `<div id="Grid_*"` has it's own close tag.
This article a bit difference with others cause it do not wrap code in a `pre` tag instead a hidden `div`.

**Review by t7yang — CHANGES_REQUESTED** [review]

File: files/zh-cn/learn/css/css_layout/grids/index.html (hunk 3 above)
Comment (meowmeowmeowcat) [reply files/zh-cn/learn/css/css_layout/grids/index.html:86]: You're right. Thanks for catching this!

**Comment by meowmeowmeowcat** [comment]
All live samples should work properly now.

**Review by t7yang — APPROVED** [review]

**Pull request merged**